tests/fsharpqa/Source/run.fs

```diff
@@ -0,0 +1,1479 @@
+           printf "%s" cmtStr
+    
+    //my $exit_str;
+    let exit_str = ""
+    //my $test_result = $exitVal;
+    let test_result = exitVal
+    
+    // # Run POSTCMD if any
+    //if (defined($ENV{POSTCMD})) {
+    do! if (not(System.String.IsNullOrWhiteSpace(envPOSTCMD))) then
+            // # Do the magic to replace known tokens in the
+            // # PRECMD/POSTCMD: for now you can write in env.lst
+            // # something like:
+            // #    SOURCE=foo.fs POSTCMD="\$FSC_PIPE bar.fs"
+            // # and it will expanded into $FSC_PIPE before invoking it
+            //$_ = $ENV{POSTCMD};
+            //s/^\$FSC_PIPE/$FSC_PIPE/;
+            //s/^\$FSI_PIPE/$FSI_PIPE/;
+            //s/^\$FSI32_PIPE/$FSI32_PIPE/;
+            //s/^\$CSC_PIPE/$CSC_PIPE/;
+            //s/^\$VBC_PIPE/$VBC_PIPE/;
+            TODO "implement replace"
+            
+            //if (RunCommand("POSTCMD",$_,1)){
+            //     $exitVal = TEST_FAIL;
+            //     $test_result = TEST_FAIL;
+            //     $exit_str .= "Fail to execute the POSTCMD. ";
+            //}
+            TODO "implement POSTCMD"
+
+            NUnitConf.skip "POSTCMD not implemented"
+        else
+            Success
+    //}
+    
+    //if (exists($ENV{SKIPTEST})) {
+    //    $exit_str = "Test Marked: SKIP using Environment Variable SKIPTEST , Tested as: ";
+    //    $exitVal = TEST_SKIPPED;
+    //} elsif ($Skip) {
+    //    my @platforms;
+    //    $Skip_platforms=~s/\s//g;
+    //    # skip all platforms if no platforms specified
+    //    if ($Skip_platforms eq "") {
+    //        $exit_str = "Test Marked: SKIP, Tested as: ";
+    //        $exitVal = TEST_SKIPPED;
+    //    }
+    //    # treat garbage comas as fatal error
+    //    elsif (!scalar(@platforms = split(/,/,$Skip_platforms))) {
+    //        $exit_str = "Expects Skip Tag Has Errors: \"$Skip_platforms\" Test Was $status_hash{$test_result}, Tested as: ";
+    //        $exitVal = TEST_FAIL;
+    //        $test_result = TEST_FAIL;
+    //    } else {  
+    //        my $platform_to_skip;
+    //        foreach my $match (@platforms) {
+    //            # treat unrecognized platform or garbage as fatal error
+    //            unless ($platform_to_skip = $Platform_Hash{$match}) {
+    //                $exit_str = "Expects Skip Tag Has Errors: \"$Skip_platforms\" Test Was $status_hash{$test_result}, Tested as: ";
+    //                $exitVal = TEST_FAIL;
+    //                $test_result = TEST_FAIL;
+    //                last;
+    //            }
+    //            # don't break here even if we match because we might run into garbage later on
+    //            elsif ($platform_to_skip == $platform) {
+    //                $exit_str = "Test Marked: SKIP, Tested as: ";
+    //                $exitVal = TEST_SKIPPED;
+    //            }
+    //        }
+    //    }
+    //}
+    TODO "implement SKIP? or it's the runner filter?"
```

Comment:
i need to check usage when i see a test using it.
I cannot find`SKIPTEST` in tests, so the first part it's unused. The run.pl support it, but it's not used by current fsharpqa tests, so useless for us, no need to implement it.
The `else` it's used to skip tests by platform, but maybe it's useless (skip test because os < vista it's useless now for example).

I added a todo to review usage. 
i'll change to make the test fails if the condition it's true, so it will fail the a specific message and we can review it